public/node-releases-data.json

```diff
@@ -1 +1 @@
-[]
```

Comment:
Also unrelated changes here

Reply:
Why we don’t put this file in gitinignore ?